features/auth/components/LoginForm.tsx

```diff
@@ -20,10 +20,16 @@ import { LoginSchema } from '../schemas/LoginSchema'
 import { Button } from '@/components/ui/button'
 import { ErrorMessage, SuccessMessage } from './ui'
 import { login } from '@/features/auth/actions'
+import { useSearchParams } from 'next/navigation'
 
 type LoginFormData = z.infer<typeof LoginSchema>
 
 export const LoginForm = () => {
+	const searchParams = useSearchParams()
+	const urlError =
+		searchParams.get('error') === 'OAuthAccountNotLinked'
```

Comment:
zrób sobie wyżej zmienną const AUTH_ACCOUNT_NOT_LINKED_ERROR_CODE = "OAuthAccountNotLinked"
i zrób przyrównanie do niej. Jeżeli jest używana gdzieś w innym miejscu aplikacji może być w shared, ale jeżeli nie jest to zrób tego consta w tym pliku